juju/juju
# feat(migrations): wire object store getter for import/export operations

Before this patch, the object store getter was not available for migration operations (as registered in
domain/modelmigration/import|export.go), it was only available for the domain services.

Since a following patch will include new functionality on the charm domain that will require having access to an object store, and it will be passed as argument to the service constructor, it's better to have it prepared and already wired and available on the migration operations as well.





## QA steps

The object store getter is wired for migrations but isn't still injected to any actual import/export operations (will come in a future PR), bootstrapping and migrating a model should test the path though:
```
juju bootstrap lxd src 
juju add-model m
juju deploy ubuntu
juju bootstrap lxd tgt
juju switch src:admin/m
juju migrate m tgt
```
No errors should appear in the logs and model should be correctly migrated to the `tgt` controller.


## Links


**Jira card:** JUJU-6866
